tests/conftest.py

```
@@ -87,6 +88,99 @@ def counting(self, name):
+        return []  # [session] extra absent — nothing could have been started.
+
+    swept: list[Path] = []
+    for conn in connection_files:
+        try:
+            if is_session_running(conn, timeout=2.0):
+                stop_session(conn, timeout=5.0)
+                swept.append(conn)
+        except Exception:
+            # A cleanup sweep must never fail the run it is cleaning up after;
+            # a kernel we cannot reach is already as good as gone.
+            continue
```

Ruff (0.16.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)